src/main/resources/logback.xml

```diff
@@ -0,0 +1,23 @@
+<configuration scan="true" >
+
+    <property name="LOGFILE" value="${user.home}/logs/ds-cumulus-export.log" />
```

Comment:
Is the use of `${user.home}/logs/` as log-destination an established practise at Vestbredden? I am not a big fan of putting things outside of the git-controlled folder and would prefer `logs`, which in effect would be `/home/te/projects/ds-cumulus-export/logs/` for me.

Reply:
Grundlæggende ja, det er stedet som log-filer typisk har været skrevet når ting er leveret her fra "vestbredden". Altså i en test/stage/prod situation, og ikke ved lokal udvikling.
Jeg antager at dit /home/te/projects/ds-cumulis-export er dit udtjekkede repository og at du forventer at finde log filer under logs der under udviklingsbrug. 
src/main/resources er til den færdige pakke, til testformål kan der droppes en under src/test/resources som gemmer log-filer et andet sted. Alternativt til test-formål logbacks default: log på debug niveau til stdout.

Reply:
Established practice beats personal preferences. Logs in user.home.